erpnext/accounts/utils.py

```
@@ -2717,3 +2720,768 @@ def build_qb_match_conditions(doctype, user=None) -> list:
+				indicator="orange",
+			)
+
+
+def _validate_serial_and_batch_fields(doc):
+	try:
+		if hasattr(doc, "validate_standalone_serial_nos_customer"):
+			doc.validate_standalone_serial_nos_customer()
+	except frappe.ValidationError:
+		raise
+	except Exception:
+		pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)